### ISSUE TITLE
Loading state doesn't seem to be fully working

### ISSUE THREAD

**Issue by tchaton** [lessw2020/Ranger-Deep-Learning-Optimizer#19]
To save : 'optimizer' : optimizer.state_dict()

optimizer.load_state_dict(checkpoint['optimizer'])

However, I have the impression restarting the training always bring the accuracy down and then it recovers.

Best,
Thomas Chaton>
